client/app/scripts/selectors/nodes-chart.js

```diff
@@ -0,0 +1,32 @@
+import { createSelector } from 'reselect';
+import { Map as makeMap } from 'immutable';
+
+
+const allNodesSelector = state => state.get('nodes');
+
+export const nodesSelector = createSelector(
+  [
+    allNodesSelector,
+  ],
+  allNodes => allNodes.filter(node => !node.get('filtered'))
+);
+
+
+export const nodeAdjacenciesSelector = createSelector(
```

Comment:
With the removal of `returnPreviousRefIfEqual()` when creating the selector, this will always need to create a new node map.

Also, this selector has now merged the adjacency data (id + adjacency) with the other node metadata like metrics. The previous split was to make sure that the layout engine only gets called if the adjacency data changed by being dependent only on the adjacency selector.